### PR TITLE
change cli

### PR DESCRIPTION
**Pull request by Ad96el**

### CODE REVIEW

File: docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This greatly simplifies the setup.
 You only need to start the Docker image:
 
 ```bash
-docker run --rm -it -p 9944:9944 -p 9933:9933 kiltprotocol/mashnet-node:latest --dev --ws-external --rpc-external
+docker run --rm -it -p 9944:9944 -p 9933:9933 kiltprotocol/mashnet-node:latest --dev --rpc-external
```
Comment (ntn-x2) [inline docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md]: As far as I know only port 9944 is exposed now, right? The old WS port is not exposed anymore, so the port publishing should only have 9944 exposed.

Comment (ntn-x2) [reply docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md]: Or if only 9933 is used, 9944 is outdated.

File: docs/develop/02_chain/04_fullnode.md [hunk 2]
```diff
@@ -143,7 +141,7 @@ values={[
 ### Run an Archive Node
 
 The full node can also be started as a Docker container.
-To expose the WebSockets ensure that the `--rpc-external` and `--ws-external` flags are set.
+To expose the WebSockets ensure that the `--rpc-external` flags are set.
```
Comment (ntn-x2) [inline docs/develop/02_chain/04_fullnode.md]:
```suggestion
To expose the WebSockets ensure that the `--rpc-external` flag is set.
```

File: docs/participate/01_staking/01_become_a_collator/03_setup_node.md [hunk 3]
```diff
@@ -47,9 +47,9 @@ Exposing the RPC endpoint can be done using the following parameters:
 
 Exposing the RPC endpoint of a collator does not imply that it becomes accessible via the PolkadotJS Apps interface, because this requires a WebSocket to connect to the node.
 
-By default, the WebSocket port used by the node is configured to be `9944`, but it can be changed by specifying a different value with `--ws-port=<ws_port>`.
+By default, the WebSocket port used by the node is configured to be `9944`, but it can be changed by specifying a different value with `--rpc-port=<rpc_port>`.
 
-Connecting from a remote host to either the collator RPC endpoint or WS endpoint requires explicitly exposing those endpoints to the public with the `--rpc-external` and `--ws-external` options.
+Connecting from a remote host to either the collator RPC endpoint or WS endpoint requires explicitly exposing those endpoints to the public with the `--rpc-external`.
```
Comment (ntn-x2) [inline docs/participate/01_staking/01_become_a_collator/03_setup_node.md]:
```suggestion
Connecting from a remote host to either the collator RPC endpoint or WS endpoint requires explicitly exposing those endpoints to the public with the `--rpc-external` option.
```

File: docs/participate/01_staking/01_become_a_collator/04_session_keys.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ The `result` key is the HEX-encoded public part of the newly created session key
 <TabItem value="apps">
 
 In order to use the PolkadotJS Apps UI, the node WebSocket endpoint must be reachable.
-This can be done either by publicly exposing it with the `--ws-external` flag, which is discouraged, or by setting up an SSH tunnel for the WebSocket endpoint with `ssh -L 127.0.0.1:9933:127.0.0.1:9933 <user>@<server>`.
+This can be done either by publicly exposing it with the `--rpc-external` flag, which is discouraged, or by setting up an SSH tunnel for the WebSocket endpoint with `ssh -L 127.0.0.1:9933:127.0.0.1:9933 <user>@<server>`.
```
Comment (ntn-x2) [inline docs/participate/01_staking/01_become_a_collator/04_session_keys.md]: Is 9933 or 9944 the port we should advertise?

File: docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md (hunk 1 above)
Comment (Ad96el) [reply docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md]: Only [9944](https://github.com/paritytech/substrate/pull/13384/files#diff-79991d75ead2aa9757d97be3307d93100616d08c554e4c97e6567e6c8c625f1dL63)

File: docs/participate/01_staking/01_become_a_collator/04_session_keys.md (hunk 4 above)
Comment (Ad96el) [reply docs/participate/01_staking/01_become_a_collator/04_session_keys.md]: Only [9944](https://github.com/paritytech/substrate/pull/13384/files#diff-79991d75ead2aa9757d97be3307d93100616d08c554e4c97e6567e6c8c625f1dL63)

**Comment by Ad96el** [comment]
I apologize for the mess I created here 🙌. I shouldn't work when I'm feeling unwell. I have updated the bash commands here: [b70f7d7](https://github.com/KILTprotocol/docs/pull/268/commits/b70f7d7d016916ea2fbe66061d3c95b383651647).

Once I have confirmed that all commands are working correctly, I'll convert this draft back into a PR. This will likely happen after we have our first RC.

**Comment by Ad96el** [comment]
I verified that the RPC endpoint is available on port 9944, and confirmed that the provided bash command starts the node.

**Comment by Ad96el** [comment]
The bash commands should work.

**Review by ChrisChinchilla — APPROVED** [review]
🚀

**Pull request merged**